### DISCUSSION TITLE
"yarn run clean:build" returns message "No matches found: "appWith*".

### DISCUSSION THREAD

**Discussion by dergroncki** [vinissimus/next-translate#891] [Q&A]
When I run 
```
yarn run clean:build
```
I get
```
No matches found: "appWith*"
``` 
but if I run 
```
rm -rf lib plugin appWith* Dynamic* I18n* index _context loadNa* setLang* Trans* useT* withT* getP* getC* *.d.ts getT transC* wrapT* types formatElements
```
or just
```
rm -rf appWith*
```
no message is returned.

Does anyone have an explanation for this?

Thank you.
Michael

**Comment by crs1138** [comment]
Hi there. I've got a similar problem, perhaps I'm doing something wrong.

I'm trying to run an `yarn example:with-app-directory` (and in fact any other `example:*`) getting the same output. I'm using `yarn@3.4.1`.

Thanks for any help.

**Reply by crs1138** [reply]
I've created a PR that solves this issue: https://github.com/aralroca/next-translate/pull/1132
